.rubocop.yml

```diff
@@ -33,3 +33,5 @@ Layout/MultilineOperationIndentation:
   Enabled: false
 Naming/UncommunicativeMethodParamName:
   MinNameLength: 1
+Style/FrozenStringLiteralComment:
```

Comment:
@emilianodellacasa what this is for? Can't you just do what Rubocop is suggesting?